plugin/evm/config.go

```diff
@@ -0,0 +1,21 @@
+// (c) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package evm
+
+import (
+	"github.com/ava-labs/coreth/core/txpool/legacypool"
+	"github.com/ava-labs/coreth/plugin/evm/config"
+)
+
+// defaultTxPoolConfig uses [legacypool.DefaultConfig] to make a [config.TxPoolConfig]
+// that can be passed to [config.Config.SetDefaults].
+var defaultTxPoolConfig = config.TxPoolConfig{
```

Comment:
again I'm completely fine with directly copying default values from pool settings one by one if we don't want to define this and pass to config pkg.

Reply:
seems like a bit less duplication this way.